frontend/test/metabase-db/postgres/sandboxes.cy.spec.js

```diff
@@ -0,0 +1,171 @@
+import {
+  signInAsAdmin,
+  signOut,
+  restore,
+  addPostgresDatabase,
+  withDatabase,
+  USER_GROUPS,
+  describeWithToken,
+} from "__support__/cypress";
+
+const PG_DB_NAME = "QA Postgres12";
+const PG_DB_ID = 2;
+
+const { ALL_USERS_GROUP, DATA_GROUP, COLLECTION_GROUP } = USER_GROUPS;
+
+const sandboxed_user = {
+  first_name: "User",
+  last_name: "1",
+  email: "u1@metabase.test",
+  password: "12341234",
+  login_attributes: {
+    user_id: "1",
+    user_cat: "Widget",
+  },
+  // Because of the specific restrictions and the way testing dataset was set up,
+  // this user needs to also have access to "collections" (group_id: 4) in order to see saved questions
+  group_ids: [ALL_USERS_GROUP, COLLECTION_GROUP],
+};
+
+function createUser(user) {
+  return cy.request("POST", "/api/user", user);
+}
+
+describeWithToken("postgres > user > query", () => {
```

Comment:
`describeWithToken` should ensure that this test runs ONLY for enterprise edition (it will be skipped for OSS), even though the spec itself doesn't live inside `enterprise/` folder.